music21/stream/base.py

```diff
@@ -6830,6 +6831,27 @@ def makeAccidentals(
                 if e.tie is not None and e.tie.type != 'stop':
                     tiePitchSet.add(e.pitch.nameWithOctave)
 
+                # handle this note's ornaments' otherPitches
+                for orn in e.expressions:
+                    if not isinstance(orn, expressions.Ornament):
+                        continue
+
+                    orn.resolveOtherPitches(e.pitch)
+                    if not orn.otherPitches:
+                        continue
+
+                    orn.updateAccidentalDisplay(
+                        pitchPast=pitchPast,
+                        pitchPastMeasure=pitchPastMeasure,
+                        alteredPitches=alteredPitches,
+                        cautionaryPitchClass=cautionaryPitchClass,
+                        cautionaryAll=cautionaryAll,
+                        overrideStatus=overrideStatus,
+                        cautionaryNotImmediateRepeat=cautionaryNotImmediateRepeat,
+                        lastNoteWasTied=False)
```

Comment:
why is this always false?  There are notes tied into turns.

Reply:
There can be notes tied into the turn's upper and lower "other" notes?  The turn's "main" note was already handled, this call is only about the "other" notes.

Reply:
ah!  Yes, that makes sense!